seqr/utils/communication_utils.py

```diff
@@ -41,3 +43,23 @@ def send_welcome_email(user, referrer):
         password_token=user.password,
     )
     user.email_user('Set up your seqr account', email_content, fail_silently=False)
+
+
+def send_load_data_email(project, base_url, data):
+    email_content = """
+    Data from AnVIL workspace {namespace}/{name} needs to be loaded to seqr project <a href="{base_url}/project/{guid}/project_page">{project_name}</a> (guid: {guid})
+
+    The sample IDs to load are attached.    
+    """.format(
```

Comment:
I don't have my environment configured for `sendEmail`. I'm not sure the link is correct or not.

Reply:
I never figured out how to test email locally either. Once its on dev we can double check it, but it looks fine so I think its okay to merge it like this